cmd/image-builder/README.md

```diff
@@ -174,3 +174,14 @@ The file must be in the format of `key=value` pairs, separated by newlines.
 Image Builder uses the ADO `oci-image-builder` pipeline as a build backend,
 which means the images are built, signed, and pushed to the Google Cloud Artifact Registry in the ADO pipeline.
 Image Builder does not build images locally on GitHub runners.
+
+## Jenkins Integration
+
+Image Builder includes basic integration with a Jenkins.
+This integration allows the `image-builder` binary to gather inputs directly from the Jenkins pipeline runtime environment.
+
+- **Purpose**: This feature is specifically designed for integration with the SRE Jenkins pipelines and **does not provide a simplified or
+  enhanced workflow experience** comparable to the GitHub workflow approach.
+- **Build Report**: A dedicated flag (`--build-report-path`) has been added to the `image-builder` binary, enabling it to write a build
+  report generated by the ADO pipeline to a specified file. This functionality is intended specifically for use in Jenkins.
+- **Usage Limitation**: This feature is intended exclusively for the SRE team. It is not supported for use by other teams or projects.
```

Comment:
This could also be a note at the beginning of the section.

Reply:
Converted to note.